vector/src/main/java/im/vector/services/EventStreamService.java

```diff
@@ -1640,9 +1633,11 @@ else if (null == CallsManager.getSharedInstance().getActiveCall()) {
      * @param callId  the callId
      */
     public void displayCallInProgressNotification(MXSession session, Room room, String callId) {
+        final NotificationManagerCompat nm = NotificationManagerCompat.from(EventStreamService.this);
```

Comment:
the call notification must be in foreground to keep the call alive.
FLAG_NO_CLEAR is not enough.